libbeat/outputs/logstash/logstash.go

```diff
@@ -46,7 +46,7 @@ func makeLogstash(
 	cfg *common.Config,
 ) (outputs.Group, error) {
 	if !cfg.HasField("index") {
-		cfg.SetString("index", -1, beat.Beat)
+		cfg.SetString("index", -1, beat.IndexPrefix)
```

Comment:
We should separate config unpacking into a separate function and create tests (file `config_test.go`) for config unpacking edge cases.

common pattern:

```
func readConfig(in *common.Config) (Config, error) {
   ...
}
```